### PR TITLE
add href support to expedition-button

### PR DESCRIPTION
**Pull request by toddmedema**
closes https://github.com/Fabricate-IO/expedition-app/issues/73

closes https://github.com/Fabricate-IO/expedition-app/issues/37

### CODE REVIEW

File: app/elements/quest-tester.html [hunk 1]
```diff
@@ -87,6 +87,15 @@
       _openURLChooser: function() {
         this.$.urlchooser.open();
       },
+      _openBrowserDeviceEmulation: function() {
```
Comment (smartin015) [inline app/elements/quest-tester.html]: This isn't very sustainable. Suggest adding an "href" attribute to the expedition-button template.

**Comment by toddmedema** [comment]
@smartin015 good call, updated to the more sustainable system

File: app/elements/base/expedition-button.html [hunk 2]
```diff
@@ -73,7 +73,18 @@
       is: 'expedition-button',
       properties: {},
       _onTap: function(e) {
-        this.fire('tap');
+        var target = e.target;
+        while (target.nodeName.toLowerCase() !== 'body' && target.nodeName.toLowerCase() !== 'expedition-button') {
+          target = target.parentNode;
+        }
+
+        if (target.getAttribute('href')) {
```
Comment (smartin015) [inline app/elements/base/expedition-button.html]: Interestingly, this'll open an href on the <body> element.

Comment (toddmedema) [inline app/elements/base/expedition-button.html]: @smartin015 yep - even better, realized that we can just check for expedition button - the alternative is that it keeps looping all the way up to the root HTML element, and then parentNode is undefined. Check out the new push

**Comment by toddmedema** [comment]
removing not ready to merge label, ready for final review

**Comment by smartin015** [comment]
LGTM

**Pull request merged**